src/foam/core/ImportsExports.js

```diff
@@ -250,11 +250,16 @@ foam.CLASS({
       name: 'imports',
       adaptArrayElement: function(o) {
         if ( typeof o === 'string' ) {
-          var a        = o.split(' as ');
-          var key      = a[0];
+          // Matches strings in the form [(javatype)] (name[?]) [as (name)]
```

Comment:
Why does this appear in this PR. Isn't this already merged?

Reply:
Yes it is, looks like a master update by me caused this to be a commit for us? ... will look to see what I can do about this.